Examples/Tests/nodal_electrostatic/analysis_3d.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+import numpy as np
+
+sys.path.insert(1, '../../../../warpx/Regression/Checksum/')
+import checksumAPI
+
+# this will be the name of the plot file
+fn = sys.argv[1]
+
+# check that the maximum chi value is small
+fname = 'diags/reducedfiles/ParticleExtrema_beam_p.txt'
+chi_max = np.loadtxt(fname)[:,19]
+assert(np.sum(chi_max<2.e-8) == len(chi_max))
```

Comment:
I think this is equivalent, right? May be easier to read, up to your preference:
```suggestion
assert np.all(chi_max < 2e-8)
```